### PR TITLE
fix(core): improve module not found error 

### PR DESCRIPTION
**Pull request by gion**
Update: https://github.com/nuxt/nuxt.js/pull/7267#issuecomment-634946919

## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

This PR adds logs the original error to the output in order to aid debugging.

In my case, I was chasing a __FATAL__ _"Module x not found"_ error and it was really hard to debug, since the underlying cause was stripped from the logs.  

Before this commit:
Screenshot 2020-04-24 at 13 31 43

After this commit:
Screenshot 2020-04-24 at 13 38 39

In my particular case, this kind of message would've saved me at least an hour of debugging ⏳.

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/core/src/module.js [hunk 1]
```diff
@@ -160,6 +160,9 @@ export default class ModuleContainer {
 
         message = message.replace(/{name}/g, src)
 
+        // add this extra warning that points to the original error for debuging purposes
+        consola.warn('originalError:', error)
+
         if (this.options._cli) {
           throw new Error(message)
```
Comment (pi0) [inline packages/core/src/module.js:167]: Nice improvement! Maybe here instead we can throw original error (stack + message) but only refine error.message to add additional hints?

Comment (gion) [reply packages/core/src/module.js:167]: I didn't dive in too deep to see what conventions do you use or how are you guys handling errors in general, so I guess, there is a lot of room for improvement :). 
Also, I think that there are other cases as well where the original error is shadowed and would be nice to be exposed.

If you feel that this small improvement needs further discussion/refactoring, maybe let's create an issue and move this fix there.  

Meanwhile, it's up to you if this fix can be merged.

Cheers ✌️

Comment (pi0) [reply packages/core/src/module.js:167]: Indeed. Thanks for bringing it to top. I will push changes to this PR if is okay ☺️

Comment (gion) [reply packages/core/src/module.js:167]: Sounds awesome! Thanks!

**Comment by pi0** [comment]
**Update:**

Non-exsistent module (without alias or local import):

image

If `foo.js` trys to import `bar.js` which is not existent:

image

**Review by pi0 — APPROVED** [review]

**Comment by gion** [comment]
awesome 🥳

**Pull request merged**